client/src/components/ItemCard.tsx

```diff
@@ -35,13 +36,23 @@ const useStyles = makeStyles(() => ({
     borderRadius: "50%",
     display: "flex",
     justifyContent: "center",
+    background: "white",
   },
 }));
 
 export const ItemCard = ({ item, onClick }: Props) => {
   const handleClick = () => {
     onClick(item);
   };
+  const [isInstrument, setInstrument] = useState(false);
+  // check if the image is an instrument
+  useEffect(() => {
```

Comment:
I believe you can do this in one line:

`setInstrument(item.image.includes("instruments"));`

Reply:
True! Changed them to:
` useEffect(() => {
    setInstrument(item.image.includes("instruments"));
  }, [item.image]);`